internal/command/run_create.go

```diff
@@ -19,12 +19,29 @@ type CreateRunCommand struct {
 	Workspace              string
 	ConfigurationVersionID string
 	Message                string
+	TargetAddrs            []string
 
 	PlanOnly  bool
 	IsDestroy bool
 	SavePlan  bool
 }
 
+// flagStringSlice is a flag.Value implementation which allows collecting
+// multiple instances of a single flag into a slice. This is used for flags
+// such as -target=aws_instance.foo and -var x=y.
+type flagStringSlice []string
+
+var _ flag.Value = (*flagStringSlice)(nil)
+
+func (v *flagStringSlice) String() string {
+	return strings.Join(*v, ",")
+}
+func (v *flagStringSlice) Set(raw string) error {
+	*v = append(*v, raw)
+
+	return nil
+}
```

Comment:
Additional test case(s) to account for: 
`--target=aws_resourrce.test_2,aws_resource.test_3`
`--target=aws_resource.test --target=aws_resourrce.test_2,aws_resource.test_3`

```suggestion
func (v *flagStringSlice) Set(raw string) error {
	targetSegments := strings.Split(raw, ",")
	*v = append(*v, targetSegments...)

	return nil
}
```